src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -39,7 +39,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <CrossCompileRid />
       <CrossCompileRid Condition="'$(_hostOS)' != '$(_originalTargetOS)' or '$(_hostArchitecture)' != '$(_targetArchitecture)'">$(RuntimeIdentifier)</CrossCompileRid>
 
-      <CrossCompileArch />
+      <CrossCompileArch>$(_targetArchitecture)</CrossCompileArch>
```

Comment:
What is this going to do to conditions that check for empty CrossCompileArch?

Reply:
Ah, right. We need:
```suggestion
      <CrossCompileArch Condition="'$(_hostArchitecture)' != '$(_targetArchitecture)'">$(_targetArchitecture)</CrossCompileArch>
```